### ISSUE TITLE
cant compile html5

### ISSUE THREAD

**Issue by maxiz1225** [KadeDev/trickster#13]
having trouble compiling the game to html5 

im getting errors in main.hx and thread.hx errors

**Comment by GithubPerson2021** [comment]
did you install the acute thiing

**Comment by Stefannag** [comment]
> did you install the acute thiing

What 'actue thing' do you mean?

**Comment by maxiz1225** [comment]
> did you install the acute thiing

yeah but i still cant compile it

**Comment by Stefannag** [comment]
> > did you install the acute thiing
> 
> yeah but i still cant compile it

Can you please tell me what do you installed? I have some errors when compiling to HTML5, and maybe that info would help me. By the way, if you want, we can talk about this subject on discord.

**Comment by maxiz1225** [comment]
> > > did you install the acute thiing
> > 
> > 
> > yeah but i still cant compile it
> 
> Can you please tell me what do you installed? I have some errors when compiling to HTML5, and maybe that info would help me. By the way, if you want, we can talk about this subject on discord.

i have the necessary stuff installed in haxe to compile fnf and some mods but this mod gives me errors in the thread.hx and also main.hx while looking in github forks and pull requests i was able to temporary fix the main.hx from giving me errors all i have left is thread.hx giving me errors and idk how to fix that

**Comment by KadeDev** [comment]
right, more power to you if you figure out how to compile to html5. but I don't support compiling to that target, at all. I wouldn't really know how to fix those issues.

**Comment by maxiz1225** [comment]
> right, more power to you if you figure out how to compile to html5. but I don't support compiling to that target, at all. I wouldn't really know how to fix those issues.

kade i checked the other issues u say that cutscenes are not compatible with html5 do u know which source files are that handles the video and cutscenes i can prob remove them and that might fix the issues bc some mods that are in html5 the cutscenes are gone i might be able to do the same with this mod

**Comment by WhittyFNF** [comment]
yea if you check the updated files you'll see the video ones @maxiz1225

**Comment by TentaRJ** [comment]
> > right, more power to you if you figure out how to compile to html5. but I don't support compiling to that target, at all. I wouldn't really know how to fix those issues.
> 
> kade i checked the other issues u say that cutscenes are not compatible with html5 do u know which source files are that handles the video and cutscenes i can prob remove them and that might fix the issues bc some mods that are in html5 the cutscenes are gone i might be able to do the same with this mod

Oh yeah just delete PlayState.hx because that is where the cutscenes are.

_**More performance 🤔**_

**Comment by KadeDev** [comment]
> > right, more power to you if you figure out how to compile to html5. but I don't support compiling to that target, at all. I wouldn't really know how to fix those issues.
> 
> kade i checked the other issues u say that cutscenes are not compatible with html5 do u know which source files are that handles the video and cutscenes i can prob remove them and that might fix the issues bc some mods that are in html5 the cutscenes are gone i might be able to do the same with this mod

we'll its an external library, but you can remove the cutscene call's in PlayState.hx

**Comment by AtonyGit** [comment]
yeah html is a hard no

**Comment by s-zenmode** [comment]
i removed playstate.hx, however there was still a bug with compiling. there's something wrong with the actual webm extension thing itself. compiling in exe has no problems though. 😐

**Comment by TentaRJ** [comment]
> **i removed playstate.hx**, however there was still a bug with compiling. there's something wrong with the actual webm extension thing itself. compiling in exe has no problems though. 😐

![Man Reading Paper](https://previews.123rf.com/images/andreypopov/andreypopov1310/andreypopov131000615/23345789-contemplated-young-man-reading-paper-holding-in-hands.jpg)
